streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -247,7 +250,7 @@
     <V1, V2> KStream<K, V2> leftJoin(
             KStream<K, V1> otherStream,
             ValueJoiner<V, V1, V2> joiner,
```

Comment:
For consistency with the `KStream` interface, shouldn't we refer to the types of `ValueJoiner` as `V1, V2, R` rather than `V, V1, V2` throughout the code, @guozhangwang ?